examples/getting-started/hello-instancing-low/app.js

```diff
@@ -155,7 +155,6 @@ export default class AppAnimationLoop extends MiniAnimationLoop {
   }
 }
 
-/* global window */
```

Comment:
gatsby has a very picky linter, just getting rid of the warnings for a clean build...